src/secret-storage.ts

```diff
@@ -136,3 +140,433 @@ export interface AccountDataClient extends TypedEventEmitter<ClientEvent.Account
      */
     setAccountData: (eventType: string, content: any) => Promise<{}>;
 }
+
+/**
+ *  Application callbacks for use with {@link SecretStorage}
+ */
+export interface SecretStorageCallbacks {
+    getSecretStorageKey?: (
+        keys: { keys: Record<string, SecretStorageKeyDescription> },
```

Comment:
Can you please add docs what `string` should be here and how it relates to the `string` of the result?

Reply:
Honestly if you can figure it out you're doing better than me. The react-sdk's implementation of this method doesn't even use this parameter :(.

I'll spend some time trying to understand.